client/extensions/woocommerce/app/settings/taxes/taxes-nexus.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * External dependencies
+ */
+import { bindActionCreators } from 'redux';
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import AddressView from 'woocommerce/components/address-view';
+import {
+	areSettingsGeneralLoading,
+	getStoreLocation
+} from 'woocommerce/state/sites/settings/general/selectors';
+import Card from 'components/card';
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import {
+	fetchSettingsGeneral,
+	updateStoreAddress
+} from 'woocommerce/state/sites/settings/general/actions';
+import { getSiteTitle } from 'state/sites/selectors';
+
+class TaxesNexus extends Component {
+
+	static propTypes = {
+		site: PropTypes.shape( {
+			slug: PropTypes.string,
+		} ),
+	};
+
+	componentDidMount = () => {
+		const { site } = this.props;
+
+		if ( site && site.ID ) {
+			this.props.fetchSettingsGeneral( site.ID );
+		}
+	}
+
+	onEditAddress = ( event ) => {
+		event.preventDefault();
+		// TODO - modal up
```

Comment:
Agreed! Swapped out in https://github.com/Automattic/wp-calypso/pull/15541/commits/5552a8792688f39b535c77718f94bdaf20514ec9